Scripts/DCS-BIOS/lib/modules/Module.lua

```diff
@@ -98,6 +98,29 @@ function Module:defineFloat(identifier, arg_number, limits, category, descriptio
 	return control
 end
 
+function Module:define8BitFloatFromGetter(identifier, func, limits, category, description)
```

Comment:
Remember to add documentation, and also tests for any new control

Reply:
fixed

Reply:
Will you be adding tests to ModuleTests?

Reply:
Yes I will try.